src/datatype/ostree.rs

```diff
@@ -81,17 +82,19 @@ impl OstreePackage {
     }
 
     /// Shell out to the ostree command to install this package.
-    pub fn install(&self, token: Option<&AccessToken>) -> Result<InstallOutcome, InstallOutcome> {
+    pub fn install(&self, creds: Credentials) -> Result<InstallOutcome, InstallOutcome> {
         debug!("installing ostree package: {:?}", self);
 
-        let mut command = Command::new("sota_ostree.sh");
-        command.env("COMMIT", self.commit.clone())
-            .env("REF_NAME", self.refName.clone())
-            .env("DESCRIPTION", self.description.clone())
-            .env("PULL_URI", self.pullUri.clone());
-        token.map(|t| command.env("AUTHPLUS_ACCESS_TOKEN", t.access_token.clone()));
+        let mut cmd = Command::new("sota_ostree.sh");
+        cmd.env("COMMIT", self.commit.clone());
+        cmd.env("REF_NAME", self.refName.clone());
+        cmd.env("DESCRIPTION", self.description.clone());
+        cmd.env("PULL_URI", self.pullUri.clone());
+        creds.access_token.map(|t| cmd.env("AUTHPLUS_ACCESS_TOKEN", t.clone()));
```

Comment:
Just double checking. These env vars are only the path to the file, not the contents, correct?